### PR TITLE
Don't panic on a sink error

### PR DESCRIPTION
**Pull request by bkchr**

### CODE REVIEW

**Review by ordian — APPROVED** [review]

File: client/rpc/src/author/mod.rs [hunk 1]
```diff
@@ -215,7 +215,7 @@ impl<P, Client> AuthorApi<TxHash<P>, BlockHash<P>> for Author<P, Client>
 				Ok(watcher) => {
 					subscriptions.add(subscriber, move |sink| {
 						sink
-							.sink_map_err(|_| unimplemented!())
+							.sink_map_err(|e| log::debug!("Subscription sink failed: {:?}", e))
```
Comment (ordian) [inline client/rpc/src/author/mod.rs:218]: add a target maybe?

Comment (bkchr) [reply client/rpc/src/author/mod.rs:218]: The other logs in the crate also don't use any `target` :see_no_evil: 

But I also don't have a better target than the automatic `sc_rpc`. I could just come up with `sc-rpc` :see_no_evil:

**Review by gww-parity — APPROVED** [review]

File: client/rpc/src/author/mod.rs (hunk 1 above)
Comment (ordian) [reply client/rpc/src/author/mod.rs:218]: `sc-rpc` sounds good

**Review by andresilva — APPROVED** [review]

File: client/rpc/src/author/mod.rs (hunk 1 above)
Comment (andresilva) [reply client/rpc/src/author/mod.rs:218]: Let's do this in another PR if we really want it. None of the RPC logging calls in `client/rpc` have any target set, so if we want to change this we should update all of them.

**Pull request merged**